code/modules/cargo/packs/_packs.dm

```diff
@@ -122,3 +122,14 @@
 	name = "[purchaser]'s Materials Order"
 	src.cost = cost
 	src.contains = contains
+
+/datum/supply_pack/custom/minerals/fill(obj/structure/closet/crate/C)
+	. = ..()
```

Comment:
There is no need to call this. You can put the material into the crate in a more efficient way. 

Currently if 50 sheets are ordered the parent proc will create a for loop that will run 50 times spawning one single sheet at a time and will merge them every time which is highly inefficient.

What you can do instead is just pass that amount to the sheet constructor and create just 1 single instance.

Have a look at the parent proc we don't need it, we can do a much better job of creating the sheets ourselves

Reply:
This is true that it's inefficient, however in this case it's not very hot code so for this 5 seconds we can leave it to get this passed along, especially since in testing it's behaving as previously expected.